test/contrib/rails/apps/cache.rb

```diff
@@ -1,15 +0,0 @@
-module RedisStoreDataAccessor
```

Comment:
had to remove this one because it was obfuscating how Rails 3.x works with cache. Think it should be fine now 👍